tools/min-base-version

```diff
@@ -27,15 +27,26 @@ from glob import glob
 
 proj_dir = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 
-max_version = '0'
-for manifest in glob(os.path.join(proj_dir, 'pkg/*/manifest.json')):
+max_version = 0
+
+for manifest in glob(os.path.join(proj_dir, 'pkg/*/manifest.json.in')):
```

Comment:
The danger of this is that by adding an exotic package that depends on the newest cockpit version, we're automatically bumping the requirement on all cockpit packages that use `required_base`.
At the very least we probably need to limit the packages we look at (hardcoded or derived from the spec file / manifests).